cms/jinja2/assets/js/wagtail-editor-customisations.js

```diff
@@ -0,0 +1,15 @@
+function setInitialMinimapExpandedByDefault(){
+    if (localStorage.getItem("wagtail:minimap-expanded") === null){
+      localStorage.setItem('wagtail:minimap-expanded', true);
+    }
+  };
+  
+
+  function setInitialRichtextToolbarStickyByDefault(){
+    if (localStorage.getItem("wagtail:draftail-toolbar") === null){
+      localStorage.setItem('wagtail:draftail-toolbar', 'sticky');
+    }
+  };
+  
+setInitialMinimapExpandedByDefault();
+setInitialRichtextToolbarStickyByDefault();
```

Comment:
Looks like this file is not being formatted due to this exclusion in https://github.com/ONSdigital/dis-wagtail/blob/main/.prettierignore#L12

I suggest replacing `jinja2` with `.min.js` and `.min.css` to prevent the formatting of minified js/css. Alternatively, exclude the `third-party` folder. I don't see a reason to include all of the `jinja2` folders. HTML is already excluded.

Reply:
[de3d237](https://github.com/ONSdigital/dis-wagtail/pull/106/commits/de3d23708f50f28261cab4beba9b2bc776f33ef4)